spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/spring/data/firestore/FirestoreTemplate.java

```diff
@@ -443,4 +460,8 @@ private Object getIdValue(Object entity) {
 	public FirestoreClassMapper getClassMapper() {
 		return this.classMapper;
 	}
+
+	public DocumentReference getDocumentReference(String collection, String documentId) {
+		return this.firestore.collection(collection).document(documentId);
+	}
```

Comment:
I believe we can do without this method now. Once you remove it, the `Firestore` client library object can go away.

Reply:
Done, thank you